src/composables/useDelegates.ts

```diff
@@ -142,6 +149,43 @@ export function useDelegates(delegatesConfig: DelegatesConfig) {
     return tx;
   }
 
+  async function fetchDelegateVotesAndProposals(
+    delegates: string[],
+    space: string
+  ) {
+    const response: { votes: Vote[]; proposals: Proposal[] } =
+      await apolloQuery({
+        query: DELEGATE_VOTES_AND_PROPOSALS,
+        variables: {
+          delegates,
+          space: 'uniswap'
+        }
+      });
+
+    if (!response) return {};
+
+    const votesAndProposals: DelegatesStats = {};
+
+    delegates.forEach(delegate => {
+      votesAndProposals[delegate] = {
+        votes: [],
+        proposals: []
+      };
+    });
+
+    response.votes.forEach(vote => {
+      const delegate = vote.voter.toLowerCase();
+      votesAndProposals[delegate].votes.push(vote);
+    });
+
+    response.proposals.forEach(proposal => {
+      const delegate = proposal.author.toLowerCase();
+      votesAndProposals[delegate].proposals.push(proposal);
+    });
+
+    delegatesStats.value = votesAndProposals;
```

Comment:
Can we just store counts here

Reply:
No, we will need the data to show the full list once we have a design